--- thrift-0.5.0/lib/php/src/ext/thrift_protocol/php_thrift_protocol.cpp.orig	2010-09-28 16:07:23.000000000 +0000
+++ thrift-0.5.0/lib/php/src/ext/thrift_protocol/php_thrift_protocol.cpp	2013-06-24 20:49:23.000000000 +0000
@@ -83,7 +83,7 @@
 #include "zend_exceptions.h"
 #include "php_thrift_protocol.h"
 
-static function_entry thrift_protocol_functions[] = {
+static zend_function_entry thrift_protocol_functions[] = {
   PHP_FE(thrift_protocol_write_binary, NULL)
   PHP_FE(thrift_protocol_read_binary, NULL)
   {NULL, NULL, NULL}
