diff -up kevinmehall-pithos-d66ff7a/pithos.desktop.orig kevinmehall-pithos-d66ff7a/pithos.desktop
--- kevinmehall-pithos-d66ff7a/pithos.desktop.orig	2012-05-01 22:10:54.055197040 -0400
+++ kevinmehall-pithos-d66ff7a/pithos.desktop	2012-05-01 22:11:20.709312864 -0400
@@ -3,6 +3,6 @@ Name=Pithos
 Comment=Play music from Pandora Radio
 Categories=GNOME;AudioVideo;Player;
 Exec=pithos
-Icon=pithos
+Icon=/usr/share/pithos/media/icon.png
 Terminal=false
 Type=Application
