diff -up pithos-0.3.16/pithos.desktop.in.orig pithos-0.3.16/pithos.desktop.in
--- pithos-0.3.16/pithos.desktop.in.orig	2012-04-25 12:07:08.428110702 -0400
+++ pithos-0.3.16/pithos.desktop.in	2012-04-25 12:07:26.947148889 -0400
@@ -3,6 +3,6 @@ Name=Pithos
 Comment=Play music from Pandora Radio
 Categories=GNOME;AudioVideo;Player;
 Exec=pithos
-Icon=pithos
+Icon=/usr/share/pithos/media/icon.png
 Terminal=false
 Type=Application
